tests/baselines/reference/user/chrome-devtools-frontend.log

```diff
@@ -4868,9 +4827,9 @@ node_modules/chrome-devtools-frontend/front_end/devices/DevicesView.js(279,45):
 node_modules/chrome-devtools-frontend/front_end/devices/DevicesView.js(280,60): error TS2555: Expected at least 2 arguments, but got 1.
 node_modules/chrome-devtools-frontend/front_end/devices/DevicesView.js(283,88): error TS2555: Expected at least 2 arguments, but got 1.
 node_modules/chrome-devtools-frontend/front_end/devices/DevicesView.js(285,36): error TS2694: Namespace 'Adb' has no exported member 'PortForwardingRule'.
-node_modules/chrome-devtools-frontend/front_end/devices/DevicesView.js(286,36): error TS2345: Argument of type 'this' is not assignable to parameter of type '{ renderItem(item: T, editable: boolean): Element; removeItemRequested(item: T, index: number): void; beginEdit(item: T): Editor<T>; commitEdit(item: T, editor: Editor<T>, isNew: boolean): void; }'.
-  Type 'PortForwardingView' is not assignable to type '{ renderItem(item: T, editable: boolean): Element; removeItemRequested(item: T, index: number): void; beginEdit(item: T): Editor<T>; commitEdit(item: T, editor: Editor<T>, isNew: boolean): void; }'.
-    Property '_callback' does not exist on type '{ renderItem(item: T, editable: boolean): Element; removeItemRequested(item: T, index: number): void; beginEdit(item: T): Editor<T>; commitEdit(item: T, editor: Editor<T>, isNew: boolean): void; }'.
+node_modules/chrome-devtools-frontend/front_end/devices/DevicesView.js(286,5): error TS2322: Type 'ListWidget<any>' is not assignable to type '{ renderItem(item: T, editable: boolean): Element; removeItemRequested(item: T, index: number): void; beginEdit(item: T): Editor<T>; commitEdit(item: T, editor: Editor<T>, isNew: boolean): void; } & { ...; }'.
+  Type 'ListWidget<any>' is not assignable to type '{ renderItem(item: T, editable: boolean): Element; removeItemRequested(item: T, index: number): void; beginEdit(item: T): Editor<T>; commitEdit(item: T, editor: Editor<T>, isNew: boolean): void; }'.
+    Property 'renderItem' is missing in type 'ListWidget<any>'.
```

Comment:
exposed by excess property check removal.

Wow, this is a bad one:

```js
var UI = {}
UI.ListWidget = class { };

UI.ListWidget.Delegate = function() {};
UI.ListWidget.Delegate.prototype = {
  renderItem() {},
};

/** @type {UI.ListWidget} */
var l = new UI.ListWidget();
```

The type of `l` isn't `UI.ListWidget`, but `{ renderItem() {} } & { renderItem() {} }` -- the structural type of UI.ListWidget.Delegate, duplicated.

Filed #26496